books/RayTracingTheRestOfYourLife.html

```diff
@@ -1249,15 +1266,16 @@
     Generating importance-sampled points on the unit sphere]
 </div>
 
-The analytic answer (if you remember enough advanced calc, check me!) is $\frac{4}{3} \pi$, and the
-code above produces that. Next, we are ready to apply that in ray tracing!
-
-The key point here is that all the integrals and probability and all that are over the unit sphere.
-The area on the unit sphere is how you measure the directions. Call it direction, solid angle, or
-area -- it’s all the same thing. Solid angle is the term usually used. If you are comfortable with
-that, great! If not, do what I do and imagine the area on the unit sphere that a set of directions
-goes through. The solid angle $\omega$ and the projected area $A$ on the unit sphere are the same
-thing.
+The analytic answer is $\frac{4}{3} \pi$ (if you remember enough advanced calc, check me!), and the
+code above produces that. The key point here is that all of the integrals and the probability and
+everything else is over the unit sphere. The area on the unit sphere is how you measure direction in
+3D. Call it direction, solid angle, or area -- it’s all the same thing. Solid angle is the term that
```

Comment:
“The area on the unit sphere is how you measure direction in 3D.” I don't understand this at all. Can you expand on this?